+bids/+internal/get_metadata.m

```diff
@@ -1,8 +1,17 @@
 function meta = get_metadata(filename, pattern)
+  %
   % Read a BIDS's file metadata according to the inheritance principle
-  % FORMAT meta = bids.internal.get_metadata(filename, pattern)
-  % filename    - name of file following BIDS standard
-  % pattern     - regular expression matching metadata file
+  %
+  % USAGE::
+  %
+  %    meta = bids.internal.get_metadata(filename, pattern)
+  %
+  % :param filename: name of file following BIDS standard
+  % :type  filename: string
+  % :param pattern:  regular expression matching metadata file
```

Comment:
The docstring is a little bit confused. Reading from code, filemane must be datafile, and pattern must contain `%s`. This must appear in docstring.

Reply:
Oh nice. I am slowly trying to improve the "help" sections and reformating them to use sphynx in the end to generate the doc. But I am clearly lagging behind in keeping that up to date.